lte/gateway/c/core/oai/tasks/amf/nas5g_message.cpp

```diff
@@ -307,10 +311,11 @@ int nas5g_message_encode(
       }
     }
     OAILOG_INFO(
-        LOG_AMF_APP, "Encoded msg(nas5g) id: [%x]-name [%s]",
+        LOG_AMF_APP, "Encoded msg(nas5g) id: [%lu]-name [%s]",
```

Comment:
this should be `%u`?

Reply:
@ardzoht, 

the actual intention was to print hex value for message_type.I have reverted back to hex print.

000465 Mon Jan 24 11:19:10 2022 7F76CE0E1700 INFO  AMF-AP tasks/amf/nas5g_message.cpp     :0327    Encoded msg(nas5g) id: [**56**]-name [AUTH_REQUEST].